src/operator/cast-inl.h

```diff
@@ -0,0 +1,152 @@
+/*!
```

Comment:
Oh. It is still included from CaffeDataIter.cc. I will remove that include, then.